Source/Utils/WarpXUtil.cpp

```diff
@@ -567,6 +567,22 @@ void ReadBCParams ()
                 amrex::Abort(" PEC boundary not implemented for PSATD, yet!");
             }
         }
+        if (WarpX::field_boundary_lo[idim] == FieldBoundaryType::Absorbing_SilverMueller ||
+            WarpX::field_boundary_hi[idim] == FieldBoundaryType::Absorbing_SilverMueller){
+            AMREX_ALWAYS_ASSERT_WITH_MESSAGE(
+#ifndef WARPX_DIM_RZ
+                (WarpX::field_boundary_lo[0] == FieldBoundaryType::Absorbing_SilverMueller)&&
+#endif
+                (WarpX::field_boundary_hi[0] == FieldBoundaryType::Absorbing_SilverMueller)&&
+                (WarpX::field_boundary_lo[1] == FieldBoundaryType::Absorbing_SilverMueller)&&
+                (WarpX::field_boundary_hi[1] == FieldBoundaryType::Absorbing_SilverMueller)
+#if (AMREX_SPACEDIM==3)
+                &&
+                (WarpX::field_boundary_lo[2] == FieldBoundaryType::Absorbing_SilverMueller)&&
+                (WarpX::field_boundary_hi[2] == FieldBoundaryType::Absorbing_SilverMueller)
+#endif
+                , " The current implementation requires silver-mueller boundary condition to be applied at all boundaries!");
```

Comment:
@RemiLehe I added a temporary abort statement here ensuring that silver-mueller is selected on all valid boundaries if selected for one -- such that it is aligned with the current implementation